nupic/research/frameworks/dendrites/functional/apply_dendrites.py

```diff
@@ -114,12 +130,12 @@ def dendritic_absolute_max_gating_2d(y, dendrite_activations):
               (b, c, h, w) where the axes represent the batch, channel, height, and
               width dimensions respectively)
     :param dendrite_activations: the dendrite activation values (a torch tensor
-                                 with shape (b, c) where the axes represent the
-                                 batch and channel dimensions, respectively)
+                                 with shape (b, c, d) where the axes represent the
+                                 batch, channel, and dendrite dimensions, respectively)
```

Comment:
Same as above (i.e., "dendrite")

Reply:
Updated to use "number of units" and "number of segments".